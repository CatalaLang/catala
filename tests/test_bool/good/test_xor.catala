## [Test all 'xor' combinations]

```catala
new scope TestXor :
  param t_xor_t content bool
  param t_xor_f content bool
  param f_xor_t content bool
  param f_xor_f content bool

scope TestXor :
  def t_xor_t := true xor true
  def f_xor_t := false xor true
  def t_xor_f := true xor false
  def f_xor_f := false xor false
```
