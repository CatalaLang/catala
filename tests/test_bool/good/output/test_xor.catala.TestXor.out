[RESULT] Computation successful! Results:
[RESULT] f_xor_f = false
[RESULT] f_xor_t = true
[RESULT] t_xor_f = true
[RESULT] t_xor_t = false
